packages/yew/src/html/conversion/into_prop_value.rs

```diff
@@ -182,10 +183,13 @@ macro_rules! impl_into_prop {
 }
 
 // implemented with literals in mind
-impl_into_prop!(|value: &'static str| -> String { value.to_owned() });
+impl_into_prop!(<'a> |value: &'a str| -> String { value.to_owned() });
```

Comment:
I'm not a fan of this conversion, but since we were already doing it for `'static`, this isn't much worse either

Reply:
Normally it should be done for AttrValue anyway. I guess those are here for backward compatibility more than anything